tests/test-a8c-files-utils.php

```diff
@@ -1,6 +1,48 @@
 <?php
 
 class VIP_Go_A8C_Files_Utils_Test extends WP_UnitTestCase {
```

Comment:
These tests are technically meant for the `A8C_Files_Utils` class. Should we have a new testcase class for our new tests?

Reply:
Right, good point, I'll split them off.